deploy/02_clusterrole.yaml

```diff
@@ -28,14 +28,18 @@ rules:
   - events
   verbs:
   - "*"
+- apiGroups:
+  - scheduling.k8s.io
+  resources:
+  - priorityclasses
+  verbs:
+  - get
+  - watch
+  - list
 - apiGroups:
   - apps
   resources:
   - deployments
   - replicasets
   verbs:
   - "*"
-- apiGroups: ["batch", "extensions"]
-  resources: ["jobs", "cronjobs"]
-  verbs:
-  - "*"
```

Comment:
this looks like an indepenent change

Reply:
This is cleaning old roles, we're not using. Unlike upstream which relies on cronjob to run descheduler, we have an operator that handles that for us.

Reply:
Technically the upstream can also run the descheduler as a control-loop but recommends running it as a cronjob. We chose to deploy it as a control-loop in the operator since cronjobs aren't yet GA (though this operator did deploy it as a cronjob previously, before we recently refactored it).

Once cronjobs are GA, I think it would be good to add that option back into our operator. But obviously we don't need this role until then

Reply:
Anything under `deploy` is just for development. Not going into production.

Reply:
bad style to mix cleanup and the things described in the commit.